### PR TITLE
tiny improvements

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/js-history/infix-operators-are-left-associative.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/js-history/infix-operators-are-left-associative.md?ref=2d76d89d3d955329ebe48212ef1b974fca208e6b)

File: javascript/core/js-history/infix-operators-are-left-associative.md [hunk 1]
```diff
@@ -71,11 +50,11 @@ var x = a - (b - c);
...
-var b = 4;
-var c = 12;
+let a = 20;
+let b = 4;
+let c = 12;
 
-var x = a - (-b * c);
+let x = a - (-b * c);
 console.log(x);
 ```
```
Comment (Stefan-Stojanovic) [inline javascript/core/js-history/infix-operators-are-left-associative.md]: Revision question should be:

The processing of operations starts with the ??? operator if no brackets are used.

Missing a "the"

Comment (Stefan-Stojanovic) [reply javascript/core/js-history/infix-operators-are-left-associative.md]: I wasn't able to find how to put the comment on the revision question part, so I placed it as close as I could.

File: javascript/core/js-history/infix-operators-are-left-associative.md [hunk 2]
```diff
@@ -91,7 +70,7 @@ console.log(x);
 ---
 ## Revision
 
-The processing of operations starts with ??? operator if no brackets are used.
+The processing of operations starts with the ??? operator if no brackets are used.
```
Comment (kapnobatai137) [inline javascript/core/js-history/infix-operators-are-left-associative.md]:
```suggestion
The processing of operations starts with the ??? operator if no parentheses are used.
```

**Pull request merged**